File: Java/1006.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner scanner = new Scanner(System.in);
		
		float a = scanner.nextFloat();
		float b = scanner.nextFloat();
		float c = scanner.nextFloat();
		
        System.out.println(String.format("MEDIA = %.1f", ((a * 2) + (b * 3) + (c * 5)) /10)); 
	}
}

File: Kotlin/1006.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`);
    val a = scanner.nextDouble();
    val b = scanner.nextDouble();
    val c = scanner.nextDouble();
    println("MEDIA = %.1f".format((a * 2 + b * 3 + c * 5) / 10))
}